x-pack/test/token_api_integration/auth/session.js

```diff
@@ -5,6 +5,7 @@
  */
 
 import request from 'request';
+import expect from 'expect.js';
```

Comment:
Need to merge master and rewrite this import before merging

Reply:
Will do, thanks!